etna/transforms/feature_selection/feature_importance.py

```diff
@@ -193,3 +232,19 @@ def _fit(self, df: pd.DataFrame) -> "MRMRFeatureSelectionTransform":
             atol=self.atol,
         )
         return self
+
+    def params_to_tune(self) -> Dict[str, "BaseDistribution"]:
```

Comment:
We can also try to tune `relevance_aggregation_mode` and `redundancy_aggregation_mode`. I met a bug #1249 during trying it.